.circleci/config.yml

```diff
@@ -16,7 +16,505 @@ aliases:
     branches:
       only: master
 
+executors:
+  grafana-build:
+    docker:
+      - image: grafana/build-container:1.2.13
+  grafana-publish:
+    docker:
+      - image: grafana/grafana-ci-deploy:1.2.3
+  docker:
+    machine:
+      image: circleci/classic:201808-01
+  cloud-sdk:
+    docker:
+      - image: google/cloud-sdk
+
 jobs:
+  install-grabpl:
+    description: Install the Grafana Build Pipeline tool
+    executor: grafana-build
+    steps:
+      - run:
+          name: Clone repo
+          command: |
+            mkdir -p ~/.ssh
+            echo 'github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==' >> ~/.ssh/known_hosts
+            git clone git@github.com:grafana/build-pipeline.git
+      - run:
+          name: Install Grafana Build Pipeline
+          command: |
+            cd build-pipeline
+            go build -o ../bin/grabpl ./cmd/grabpl
+      - persist_to_workspace:
+          root: .
+          paths:
+            - bin/grabpl
+
+  build-backend:
+    description: "Build a certain variant of Grafana back-end binaries"
+    parameters:
+      edition:
+        type: string
+      variant:
+        type: string
+    executor: grafana-build
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - checkout
+      - run:
+          name: CI job started
+          command: ./scripts/ci-job-started.sh
+      - run:
+          name: Build Grafana backend
+          # TODO: Don't use $CIRCLE_TAG for master
```

Comment:
@marefr It's because of an inconsistency versus how the original master pipeline performs builds - I haven't got around to figuring it out yet. It shouldn't be a blocker from merging, since we need to do more testing and fixing anyway before considering this new pipeline complete.

Reply:
Okay, please disregard it